[2025-02-12T01:26:39.394+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T01:26:39.431+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-12T01:18:41.433292+00:00 [queued]>
[2025-02-12T01:26:39.455+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-12T01:18:41.433292+00:00 [queued]>
[2025-02-12T01:26:39.456+0000] {taskinstance.py:2866} INFO - Starting attempt 4 of 4
[2025-02-12T01:26:39.467+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_permissions_dag> on 2025-02-12 01:18:41.433292+00:00
[2025-02-12T01:26:39.472+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2237) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-12T01:26:39.473+0000] {standard_task_runner.py:72} INFO - Started process 2238 to run task
[2025-02-12T01:26:39.474+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_permissions_dag', 'manual__2025-02-12T01:18:41.433292+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpdbgr30i7']
[2025-02-12T01:26:39.475+0000] {standard_task_runner.py:105} INFO - Job 303: Subtask assign_permissions_dag
[2025-02-12T01:26:39.510+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-12T01:18:41.433292+00:00 [running]> on host 3f11f16c6f4b
[2025-02-12T01:26:39.713+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_permissions_dag' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T01:18:41.433292+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-12T01:18:41.433292+00:00'
[2025-02-12T01:26:39.716+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-12T01:26:39.722+0000] {logging_mixin.py:190} INFO - current_owner forhello_world_dag = dados
[2025-02-12T01:26:39.722+0000] {dag.py:230} INFO - Permission for DAG hello_world_dag is already correctly assigned to dados
[2025-02-12T01:26:39.723+0000] {logging_mixin.py:190} INFO - current_owner forhello_world_dag = data-team
[2025-02-12T01:26:39.723+0000] {dag.py:230} INFO - Permission for DAG hello_world_dag is already correctly assigned to dados
[2025-02-12T01:26:42.760+0000] {dag.py:248} INFO - Assigned permissions to dados for DAG hello_world_dag
[2025-02-12T01:26:45.760+0000] {dag.py:248} INFO - Assigned permissions to data-team for DAG manage_***_roles
[2025-02-12T01:26:48.513+0000] {dag.py:248} INFO - Assigned permissions to data-team for DAG manage_***_roles
[2025-02-12T01:26:48.513+0000] {logging_mixin.py:190} INFO - current_owner formanage_***_roles = data-team
[2025-02-12T01:26:48.513+0000] {dag.py:230} INFO - Permission for DAG manage_***_roles is already correctly assigned to data-team
[2025-02-12T01:26:48.513+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T01:26:48.522+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-12T01:26:48.522+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=assign_permissions_dag, run_id=manual__2025-02-12T01:18:41.433292+00:00, execution_date=20250212T011841, start_date=20250212T012639, end_date=20250212T012648
[2025-02-12T01:26:48.559+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T01:26:48.572+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T01:26:48.573+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
